dotCMS/src/main/java/com/dotmarketing/portlets/workflows/model/MultiUserReferenceParameter.java

```diff
@@ -0,0 +1,113 @@
+package com.dotmarketing.portlets.workflows.model;
+
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.Role;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UtilMethods;
+import com.liferay.util.Validator;
+import java.util.StringTokenizer;
+
+/**
+ * This type of workflow actionlet parameter allows content managers to enter a comma-separated list
+ * of values used to specify different system users. This class provides 3 mechanisms to specify
+ * users in you actionlet:
+ * <ol>
+ * <li>The user ID.</li>
+ * <li>The user's email address.</li>
+ * <li>The role key (this includes all the users associated to the specified role).</li>
+ * </ol>
+ *
+ * @author Jose Castro
+ * @version 4.3.0
+ * @since Jan 9, 2018
+ */
+public class MultiUserReferenceParameter extends WorkflowActionletParameter {
+
+    /**
+     * Creates a new instance of this class.
+     *
+     * @param key          The unique key for this parameter.
+     * @param displayName  The human-readable name of this parameter.
+     * @param defaultValue The optional default value of this parameter.
+     * @param isRequired   If {@code true}, the value of this parameter is required. Otherwise, set
+     *                     to {@code false}.
+     */
+    public MultiUserReferenceParameter(String key, String displayName,
+            String defaultValue, boolean isRequired) {
+        super(key, displayName, defaultValue, isRequired);
+    }
+
+    @Override
+    public String hasError(final String stringToValidate) {
+        final StringBuffer errorMsg = new StringBuffer();
+        if (UtilMethods.isSet(stringToValidate)) {
+            final StringTokenizer tokenizer = new StringTokenizer(stringToValidate, ",");
+            while (tokenizer.hasMoreTokens()) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS135)